etc/supervisor/conf.d/vta.amigocloud.com.conf

```diff
@@ -0,0 +1,53 @@
+[program:modeify]
+command=/home/ubuntu/modeify/runmodeify.sh
+directory=/home/ubuntu/modeify
+autostart=true
+autorestart=true
+stdout_logfile=/var/log/supervisor/vta_modeify.log
+redirect_stderr=true
+
+
+[program:vta_gunicorn]
+command=/srv/vta.amigocloud.com/virtual_env/bin/gunicorn -w 5 -b 0.0.0.0:8005 -t 30 app:app
+directory=/srv/vta.amigocloud.com/vtademo/vta
+user=www-data
+autostart=true
+autorestart=true
+stdout_logfile=/var/log/supervisor/vta_gunicorn.log
+redirect_stderr=true
+
+[program:vta_otp]
+command=java -Xmx6G -Xverify:none -jar /srv/vta.amigocloud.com/OTP/target/otp-1.0.0-shaded.jar --basePath /srv/vta.amigocloud.com/data/ --router default --server --port 8888 --cache /srv/vta.amigocloud.com/ned --autoReload --autoScan
+directory=/srv/vta.amigocloud.com/OTP
+user=www-data
+autostart=true
+autorestart=true
+stdout_logfile=/var/log/supervisor/vta_otp.log
+redirect_stderr=true
+startsecs=60
+stopwaitsecs=60
+stopsignal=KILL
+stopasgroup=true
+
+[program:vta_otp_monitor]
```

Comment:
:question: 
- [ ] Please find out if we are actually using this script, I think it's disabled unless `supervisor` is executing...